packages/react/src/components/DropdownMenu/DropdownMenu.mdx

```diff
@@ -11,34 +11,66 @@ import { DropdownMenu } from './';
 <Primary />
 <Controls />
 
-`anchorEl` er en referanse til elementet som dropdownen skal plasseres i forhold til.
-
-Bruk `useRef` for å få tak i elementet.
-Du må ha `aria-haspopup='menu'` på elementet som åpner dropdownen.
-Elementet som åpner popoveren bør bruke `aria-expanded=true/false` for å indikere om den er åpen eller ikke.
+## Slik bruker du `DropdownMenu`
 
 ```tsx
-const anchorEl = useRef(null);
-
-return (
-    <Button ref={anchorEl} aria-haspopup='menu' aria-expanded={open}>Button</Button>
-    <DropdownMenu anchorEl={anchorEl.current} />
-)
+import '@digdir/design-system-tokens/brand/altinn/tokens.css'; // Importeres kun en gang i appen din.
+import { DropdownMenu } from '@digdir/design-system-react';
+
+<DropdownMenu>
+  <DopdownMenu.Trigger>Trigger</DopdownMenu.Trigger>
+  <DropdownMenu.Content>
+    <DropdownMenu.Group>
+      <DropdownMenu.Item>Item</DropdownMenu.Item>
+    </DropdownMenu.Group>
+  </DropdownMenu.Content>
+</DropdownMenu>;
 ```
 
-## Icons
+## Eksempler på bruk
+
+### Kontrollert
+
+Dersom du sender inn `open`, så bruker du `DropdownMenu` kontrollert. Du kan bruke `onClose` for å få beskjed når `DropdownMenu` vil lukkes.
+
+<Canvas of={DropdownMenuStories.Controlled} />
+
+### Ikoner
+
+Du kan legge ikon rett inn i `DropdownMenu.Item`, dersom det blir mye mellomrom til kanten kan du legge på din egen klasse og endre på `padding`.
 
 <Canvas of={DropdownMenuStories.Icons} />
 
-## In portal
+### I portal
+
+Legg på `portal` for å rendere `DropdownMenu` i en portal. Dette flytter dropdownen til `body`.
 
 <Canvas of={DropdownMenuStories.InPortal} />
 
-### `Dropdown.Group`
+## Tilgjengelighet
+
+Dersom du bruker `DropdownMenu.Trigger` er det innebyd tilgjengelighet for deg, men dersom du bruker `anchorEl` må du styre tilgjengelighet selv.
+Da må du legge på `aria-expanded={true/false}` og `aria-haspopup='menu'`.
+
+### `anchorEl`
+
+_Dette vil bli fjernet i fremtiden, og vi anbefaler å bruke `DropdownMenu.Trigger` istedenfor._
+
+Du kan bruke `anchorEl` istedenfor `DropdownMenu.Trigger` for å styre hvor `DropdownMenu` skal vises, men da må du styre åpning og lukking av `DropdownMenu` selv.
+Vi anbefaler å bruke `DropdownMenu.Trigger` istedenfor `anchorEl` fordi det er bygd inn tilgjenelighetshåndtering for deg.
```

Comment:
Suggest moving this under "Slik bruker du `DropdownMenu`" and changing the title to "Alternativ"